docs/content/en/components/reaper/_index.md

```diff
@@ -143,7 +151,58 @@ spec:
     deploymentMode: SINGLE
 ```
 
-The list of available configuration options can be found in the [Reaper CRD]({{< relref "/reference/crd/k8ssandra-operator-crds-latest/#k8ssandraclusterspecreaper" >}}).
+### Deploying a control plane Reaper
+
+In order to deploy a control plane Reaper, you'll need to create the Reaper object directly:
+
+```yaml
+kind: Reaper
+metadata:
+  name: cp-reaper
+spec:
+  storageType: local
+  storageConfig:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 256Mi
+  httpManagement:
+    enabled: true
+```
+
+The _control plane_ Reaper must use a _local_ storage because there's no Cassandra cluster yet. 
+The local storage requires its own config, mostly to ensure enough disk space is allocated to Reaper. You can skip the `storageType` field, which will make the k8s cluster use whatever the default storage class is.
+It's also a good idea to enable the `httpManagement`. This will make Reaper use HTTP instead of JMX when interacting with Cassandra.
```

Comment:
```suggestion
We recommend enabling the `httpManagement` feature. This will make Reaper use HTTP instead of JMX when interacting with Cassandra.
```